pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -59,6 +60,11 @@ const (
 	maxDNSSearchListChars = 256
 )
 
+const (
+	multiQueueMinQueues = 2
+	multiQueueMaxQueues = 256
+)
+
```

Comment:
Maybe just add this to the previous const?

Reply:
I think makes it more explicit that this block of constants is used for on a different scope.

If you insist, I will join them.